src/org/thoughtcrime/securesms/util/Util.java

```diff
@@ -70,7 +70,12 @@ public static boolean isInviteURL(Uri uri) {
   }
 
   public static boolean isInviteURL(String url) {
-    return isInviteURL(Uri.parse(url));
+    try {
+      return isInviteURL(Uri.parse(url));
+    } catch (Exception e) {
+      e.printStackTrace();
+    }
+    return false;
```

Comment:
```suggestion
    return url != null && isInviteURL(Uri.parse(url));
```
according to the docs, Uri.parse() only throws NullPointerException if the given string is null, so the check could be simpler, in practice we never pass a null string here because otherwise it would not be a clickable link in message

Reply:
ah, thanks! so maybe not needed as urgently as i thought :)